packages/error/src/index.ts

```diff
@@ -2,7 +2,7 @@ import { OnRpcRequestHandler } from '@metamask/snaps-types';
 
 export const onRpcRequest: OnRpcRequestHandler = async () => {
   // eslint-disable-next-line no-new
-  new Promise((resolve) => {
+  await new Promise((resolve) => {
```

Comment:
It's intentional for this snap to throw an error out-of-band. That is what it is supposed to test.